database/types.go

```diff
@@ -188,8 +188,9 @@ type BlockBuilderEntry struct {
 	BuilderPubkey string `db:"builder_pubkey" json:"builder_pubkey"`
 	Description   string `db:"description"    json:"description"`
 
-	IsHighPrio    bool `db:"is_high_prio"   json:"is_high_prio"`
-	IsBlacklisted bool `db:"is_blacklisted" json:"is_blacklisted"`
+	Status       uint8  `db:"status"            json:"status"`
+	Collateral   string `db:"collateral_value"  json:"collateral_value"`
```

Comment:
May be worth renaming `Collateral` to `CollateralValue`, just for the sake of consistency.

Reply:
done!